### DISCUSSION TITLE
Is there any way to scrape a URL without using RAG?

### DISCUSSION THREAD

**Discussion by mark-os** [open-webui/open-webui#7192] [General]
# Feature Request

When I type # and the URL, it scrapes the URL and then performs a RAG search on it. For instance:

```
# https://qdrant.tech/documentation/fastembed/fastembed-colbert/
what can you tell me about this

> Model says it's related to movie recommendations. 
```

**Describe the solution you'd like**
I would like to scrape the entire page and talk about it with the model.

**Describe alternatives you've considered**
Copy pasting the page?

Let me know if I missed a better way to do this. Thanks.

**Comment by gaby** [comment]
@mark-os Did you find an answer, or how to use fastembed with OWUI?

**Reply by biebiep** [reply]
Just use the "Use entire document" switch as you would any file upload. Put the hashtag-url, click the "document" in the message box and put the slider to "Use entire document".
